packages/ui-components/src/components/SecretText/SecretText.component.tsx

```diff
@@ -294,75 +301,78 @@ export const SecretText = ({
   )
 }
 
-SecretText.propTypes = {
+//eslint-disable-next-line no-unused-vars
+type ValueChangeHandler = (value: string) => void
+
+export interface SecretTextProps extends Omit<React.HTMLAttributes<HTMLTextAreaElement>, "onCopy" | "onPaste"> {
```

Comment:
I wonder if needed:

```suggestion
export interface SecretTextProps extends Omit<React.HTMLAttributes<HTMLTextAreaElement>, "onCopy" | "onPaste" | "onChange" | "onBlur" | "onFocus"> {
```

Reply:
It appears that it is not mandatory, since declaration of "onChange", "onBlur" and "onFocus" conforms to one declared in React.HTMLAttributes<HTMLTextAreaElement>.
I have no strong opinion on omitting it from that interface though. Would you like me to do it?
@guoda-puidokaite